ui/DesignSystem/Modal.svelte

```diff
@@ -6,8 +6,12 @@
  LICENSE file.
 -->
 <script lang="typescript">
+  import { Emoji } from "ui/DesignSystem";
   export let dataCy: string = "";
   export let style: string = "";
+  export let emoji: string | undefined = undefined
+  export let title: string = "";
+  export let desc: string = "";
```

Comment:
Would be nice to convert all of the props to the same style.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/2082/commits/ffcd7be5937ebbb4cc2dbfc89d499da9641386a8